command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
```

Comment:
Rename 'new' function to a descriptive name to avoid confusion with the 'new' keyword in Go